databricks_cli/sdk/service.py

```diff
@@ -169,55 +169,55 @@ def list_runs(self, job_id=None, active_only=None, completed_only=None, offset=N
         if limit is not None:
             _data['limit'] = limit
         return self.client.perform_query('GET', '/jobs/runs/list', data=_data, headers=headers)
-
+    
     def get_run(self, run_id=None, headers=None):
         _data = {}
         if run_id is not None:
             _data['run_id'] = run_id
         return self.client.perform_query('GET', '/jobs/runs/get', data=_data, headers=headers)
-
+    
     def delete_run(self, run_id=None, headers=None):
         _data = {}
         if run_id is not None:
             _data['run_id'] = run_id
         return self.client.perform_query('POST', '/jobs/runs/delete', data=_data, headers=headers)
-
+    
     def cancel_run(self, run_id, headers=None):
         _data = {}
         if run_id is not None:
             _data['run_id'] = run_id
         return self.client.perform_query('POST', '/jobs/runs/cancel', data=_data, headers=headers)
-
+    
     def get_run_output(self, run_id, headers=None):
         _data = {}
         if run_id is not None:
             _data['run_id'] = run_id
         return self.client.perform_query('GET', '/jobs/runs/get-output', data=_data, headers=headers)
-
+    
     def export_run(self, run_id, views_to_export=None, headers=None):
         _data = {}
         if run_id is not None:
             _data['run_id'] = run_id
         if views_to_export is not None:
             _data['views_to_export'] = views_to_export
         return self.client.perform_query('GET', '/jobs/runs/export', data=_data, headers=headers)
-
+     
 
 class ClusterService(object):
     def __init__(self, client):
         self.client = client
 
     def list_clusters(self, headers=None):
         _data = {}
-
+    
         return self.client.perform_query('GET', '/clusters/list', data=_data, headers=headers)
-
+    
     def create_cluster(self, num_workers=None, autoscale=None, cluster_name=None, spark_version=None,
                        spark_conf=None, aws_attributes=None, node_type_id=None,
                        driver_node_type_id=None, ssh_public_keys=None, custom_tags=None,
-                       cluster_log_conf=None, init_scripts=None, spark_env_vars=None,
```

Comment:
Seems like you are removing the init_scripts param and the following variable-setting. I'm wondering this is part of what PR intends, and if so, we need to update the descriptions & possibly split the PR I think.

Reply:
once again, this was a result of autogeneration, wasn't relevant to my changes at all. Should I revert this particular function back to it's original form?